cores/nRF5/utility/SoftwareTimer.h

```diff
@@ -52,13 +52,29 @@ class SoftwareTimer
       _handle = xTimerCreate(NULL, ms2tick(ms), true, NULL, callback);
     }
 
+    void begin(uint32_t ms, TimerCallbackFunction_t callback, bool repeating)
```

Comment:
I think we should either 
- have repeating as default = true ( 1 begin API)
- have the above begin call this begin(ms, callback, true).

I am fine with either of them

Reply:
I've implemented the first suggestion.